views/help/partials/help_nutzungshilfen.hbs

```diff
@@ -10,7 +10,7 @@
 				{
 					"title": "Live-Formate",
 					"icon": "fa-desktop",
-					"src": "https://docs.dbildungscloud.de/display/SCDOK/Live-Schulungen"
+					"src": "https://docs.dbildungscloud.de/x/BosXBg"
```

Comment:
Since this occurence, and the occurence below, are the only ones left anymore, I'd save the time and work on the legacy client and leave it like it is.

Reply:
Okay less places to update it it's fine